Xamarin.Forms.Core/BindableLayout.cs

```diff
@@ -77,7 +77,7 @@ public static DataTemplate GetEmptyViewTemplate(BindableObject b)
 
 		public static void SetEmptyViewTemplate(BindableObject b, DataTemplate value)
 		{
-			b.SetValue(EmptyViewProperty, value);
+			b.SetValue(EmptyViewTemplateProperty, value);
```

Comment:
Fix for `BindableLayout.SetEmptyViewTemplate` sets `EmptyView` instead of `EmptyViewTemplate`